### PR TITLE
Updated doc for exporting CA_PATH to install KC using custom CA certificate

### PR DESCRIPTION
**Pull request by sethiyash**
- Updated doc for exporting CA_PATH to pull image and deploy kapp-controller using custom CA certificate

### CODE REVIEW

File: deploy.md [hunk 1]
```diff
@@ -133,11 +133,13 @@ Configure and run `install.sh`, which will install `kapp-controller` and `secret
     export INSTALL_REGISTRY_HOSTNAME=registry.tanzu.vmware.com
     export INSTALL_REGISTRY_USERNAME=TANZU-NET-USER
     export INSTALL_REGISTRY_PASSWORD=TANZU-NET-PASSWORD
+    export CA_PATH=/path/ca.crt
     cd $HOME/tanzu-cluster-essentials
     ./install.sh --yes
     ```
     
     Where `TANZU-NET-USER` and `TANZU-NET-PASSWORD` are your credentials for VMware Tanzu Network.
+    Here `CA_PATH` is optional and to be exported only when there is a need to pull the image and deploy kapp-conyroller using custom certificate provided.
```
Comment (renuy) [inline deploy.md:142]: remove "here"

File: deploy.md [hunk 2]
```diff
@@ -184,11 +188,12 @@ Configure and run `install.bat`, which will install `kapp-controller` and `secre
     set INSTALL_REGISTRY_USERNAME=TANZU-NET-USER
     set /p INSTALL_REGISTRY_PASSWORD=password:
     :: Interactively enter TANZU-NET-PASSWORD
-
+    set CA_PATH=/path/ca.crt
     install.bat
     ```
 
     Where `TANZU-NET-USER` and `TANZU-NET-PASSWORD` are your credentials for VMware Tanzu Network.
+    Here `CA_PATH` is optional and to be exported only when there is a need to pull the image and deploy kapp-conyroller using custom certificate provided.
```
Comment (renuy) [inline deploy.md:196]: type,  kapp-controller

**Comment by snneji** [comment]
@sethiyash Is this only for Cluster Essentials 1.9?

**Comment by sethiyash** [comment]
Yes @snneji, it's just for CE 1.9.

**Pull request merged**